### PR TITLE
ironSource Adapter 4.7.3.1.1.0 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by cb-jpadilla**

### CODE REVIEW

File: IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt [hunk 1]
```diff
@@ -97,28 +97,36 @@ class IronSourceAdapter : PartnerAdapter {
...
-                IronSource.setMediationType("Chartboost")
-                // IronSource leaks this Activity via ContextProvider, but it only ever leaks one
-                // Activity at a time, so this is probably okay.
-                IronSource.initISDemandOnly(
-                    context,
-                    appKey,
-                    AD_UNIT.INTERSTITIAL,
-                    AD_UNIT.REWARDED_VIDEO
-                )
+        return suspendCancellableCoroutine { continuation ->
```
Comment (bichenwang) [inline IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt:100]: we don't need to do this. ironsource is synchronous

Comment (cb-jpadilla) [reply IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt:100]: Cool. Wanted to be safe, but this clears my doubts. Thanks @bichenwang 👍🏼

File: IronSourceAdapter/build.gradle.kts [hunk 2]
```diff
@@ -75,7 +75,7 @@ dependencies {
     "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:4.0.0")
 
     // Partner SDK
-    implementation("com.ironsource.sdk:mediationsdk:7.3.1")
+    implementation("com.ironsource.sdk:mediationsdk:7.3.1.1")
```
Comment (bichenwang) [inline IronSourceAdapter/build.gradle.kts:78]: sorry, instructions unclear. we want just 7.3.1 for a 4.7.3.1.0.1 release

Comment (cb-jpadilla) [reply IronSourceAdapter/build.gradle.kts:78]: Let me modify. 😆

Comment (cb-jpadilla) [reply IronSourceAdapter/build.gradle.kts:78]: Interestingly ironSource does have 7.3.1.1

**Review by bichenwang — APPROVED** [review]
go for it!

**Pull request closed without merging**